src/content-routing.js

```diff
@@ -1,16 +1,19 @@
 'use strict'
 
-const tryEach = require('async/tryEach')
-const parallel = require('async/parallel')
 const errCode = require('err-code')
-const promisify = require('promisify-es6')
+const { messages, codes } = require('./errors')
+
+const all = require('async-iterator-all')
+const get = require('dlv')
+const pAny = require('p-any')
 
 module.exports = (node) => {
   const routers = node._modules.contentRouting || []
+  const dht = get(node, '_dht')
```

Comment:
`node` will always exist so we can drop `dlv` now right?

Reply:
yap, forgot this! Thanks for the reminder